src/npr/ont_pipeline.py

```diff
@@ -1,17 +1,22 @@
+# ont_pipeline.py
+# Helper funtions for the ONT pipeline
+# (C) 2024 Bioinformatics Core
+# Max Plank Institute for Immunobiology and Epigenetics
+
 import sys
 import os
 import re
 import glob
-import pandas as pd
-import numpy as np
-from rich import print
-from importlib.metadata import version
 import yaml
 import json
 import subprocess as sp
+import pandas as pd
+from rich import print
 from npr.communication import send_email
 from npr.snakehelper import glob2reports, get_seqdir, guppy2dorado
 
+match_html_key_val = re.compile(r'"title": "(.+?)", "value": "(.+?)"')
```

Comment:
Should this live under the read_flowcell info function ?

Reply:
could be there, I will move it